custom/icds_reports/templates/icds_reports/partials/shared_component_directives.html

```diff
@@ -4,6 +4,7 @@
   <script src="{% static 'js/directives/navigation/navigation.directive.js' %}"></script>
   <script src="{% static 'js/directives/location-filter/location-filter.directive.js' %}"></script>
   <script src="{% static 'js/directives/month-filter/month-filter.directive.js' %}"></script>
+  <script src="{% static 'js/directives/additional-filter/additional-filter.directive.js' %}"></script>
```

Comment:
surprised to see this here without it also being removed from somewhere. did you forget to delete it from `dashboard.html` and `mocha.html`?

Reply:
Should I add this to shared components? (as in which components are to be included here?). Just because other filter directives were added, I added this too. (Waiting for review)

Reply:
being here means it's used in both mobile and web dashboard (which is correct), but once that's done we should remove it explicitly from web and tests since those already import this whole file. does that make sense?

Reply:
Got it.